apps/molgenis-components/src/components/forms/InputRefList.vue

```diff
@@ -57,29 +57,27 @@
           </label>
         </div>
       </div>
-      <div v-if="canEdit">
-        <Tooltip value="New entry">
-          <RowButtonAdd
-            id="add-entry"
-            :tableId="tableId"
-            :schemaId="schemaId"
-            @update:newRow="selectNew"
-          />
-        </Tooltip>
+      <div v-if="canEdit" class="m-1">
+        <RowButtonAdd
+          id="add-entry"
+          :label="`Add new ${label}`"
+          :tableId="tableId"
+          :schemaId="schemaId"
+          @update:newRow="selectNew"
+        />
       </div>
-      <div>
-        <ButtonAlt
-          class="pl-0"
+      <div class="m-1">
```

Comment:
I think it would look better if you show the buttons next to eachother.

Reply:
The text on the buttons is very verbose, which makes putting them next to each other a bit awkward. We could reword them into something shorter, but we couldn't really come up with something. Do you have any suggestions?